docs/competitive-programming/max-number-from-array.mdx

```diff
@@ -0,0 +1,120 @@
+---
+title: Maximum number formed from numbers in array
```

Comment:
`Maximum number formed from numbers in an array`

Reply:
Done